### PR TITLE
add incident in Chicago 7/17 cop sucker punches activist

### PR DESCRIPTION
**Pull request by sprucelee**

### CODE REVIEW

File: reports/Illinois.md [hunk 1]
```diff
@@ -191,6 +191,22 @@ id: il-chicago-14
 * https://twitter.com/punishmegrant/status/1268966252707487745
 
 
+### Cop sucker punches activist | July 17th
```
Comment (emarcey) [inline reports/Illinois.md:194]: I don't think sucker punch is the most neutral description. Just punch would be better imo

**Comment by emarcey** [comment]
Also check #997 for any additional links

File: reports/Illinois.md (hunk 1 above)
Comment (sprucelee) [reply reports/Illinois.md]: Changed to just 'punch'

**Comment by sprucelee** [comment]
From what I can tell, the links in #997 are retweets. The original content is from the GoodKidsMadCity twitter account; the individual who was punched is an organizer with them.

**Pull request merged**